qualtran/bloqs/phase_estimation/phase_estimation_of_quantum_walk.ipynb

```diff
@@ -129,6 +117,44 @@
     "print(circuit)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Computing costs\n",
+    "\n",
+    "Usually, you'd define a Bloq that captures the entire phase estimation circuit, but we can use a little helper function to compute the gate counts directly from the bloqs encountered within the Cirq circuit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qualtran import Bloq\n",
+    "from qualtran.resource_counting import get_cost_value, QECGatesCost, GateCounts\n",
+    "\n",
+    "def get_qec_gates_cost_for_circuit(circuit):\n",
+    "    # Usually, you'd combine this into a bloq of its own, but we\n",
+    "    # use this helper function to add up the costs of the bloqs\n",
+    "    # found within the circuit.\n",
+    "    cost_key = QECGatesCost()\n",
+    "    costs_cache = {}\n",
+    "    total_cost = cost_key.zero()\n",
+    "    for op in circuit.all_operations():\n",
+    "        if not isinstance(op.gate, Bloq):\n",
+    "            # Skip state prep and QFT for now\n",
+    "            continue\n",
```

Comment:
Maybe add a `print("Skipping {op.gate=}")` to make sure only state prep and QFT are being skipped.

Reply:
yeah I had this when I wrote it but the stateprepchannel repr is very long and this would get printed every time